static/app/components/modals/widgetBuilder/addToDashboardModal.tsx

```diff
@@ -43,13 +72,40 @@ function AddToDashboardModal({
   }, []);
 
   function handleGoToBuilder() {
+    const pathname =
+      selectedDashboardId === 'new'
```

Comment:
in the `widgetBuilder` we have the const `const NEW_DASHBOARD_ID = 'new';` should we move it the constants and reuse here?